runtime/js/40_process.js

```diff
@@ -203,6 +205,12 @@ class ChildProcess {
   #waitPromise;
   #waitComplete = false;
 
+  #pipeFd;
+  // internal, used by ext/node
+  get _pipeFd() {
```

Comment:
This is not great. Can you think of a way how we could not expose it to user code? Let's address that in a follow up.